webview/platforms/winforms.py

```diff
@@ -350,17 +350,23 @@ def show(self):
 
         def set_window_menu(self, menu_list):
             def _set_window_menu():
+                def create_action_item(menu_line_item):
+                    action_item = WinForms.ToolStripMenuItem(menu_line_item.title)
+                    def on_click(_sender, _args, menu_line_item=menu_line_item):
+                        # Don't run action function on main thread
+                        from threading import Thread
```

Comment:
this will shadow previously imported System.Threading.Thread. Use threading.Thread instead